src/Asv.Mavlink.Shell/Tests/ExportSdrData.cs

```diff
@@ -15,20 +16,63 @@ public class ExportSdrData
     /// <param name="inputFile">-i, Input file</param>
     /// <param name="outputFile">-o, Output file</param>
     [Command("export-sdr")]
-    public void RunExportSdrData(string inputFile, string outputFile = "out.csv")
+    public async Task RunExportSdrDataAsync(string inputFile, string outputFile = "out.csv")
     {
-            using var file = File.OpenRead(inputFile);
-            using var outFile = new StreamWriter(File.OpenWrite(outputFile));
-            var data = new byte[256];
+        try
+        {
+            await using var file = File.OpenRead(inputFile);
+            if (file.Length == 0)
+            {
+                AnsiConsole.MarkupLine($"[red]Error:[/] Input file [yellow]{inputFile}[/] is empty.");
+                return;
+            }
             
-            while (file.Position < file.Length)
+            await using var outFile = new StreamWriter(File.OpenWrite(outputFile));
+
+            var buffer = new byte[256];
+
+            while (true)
             {
-                file.Read(data, 0, data.Length);
-                AsvSdrRecordDataLlzPayload payload = new AsvSdrRecordDataLlzPayload();
-                var span = new ReadOnlySpan<byte>(data);
-                payload.Deserialize(ref span);
-                outFile.WriteLine($"{payload.Alt}\t{payload.TotalAm90}\t{payload.TotalAm150}\t{payload.TotalPower}\t{payload.ClrAm90}\t{payload.ClrAm150}\t{payload.ClrPower}\t{payload.CrsAm90}\t{payload.CrsAm150}\t{payload.CrsPower}".Replace(",", "."));
+                int bytesRead = await file.ReadAsync(buffer.AsMemory());
```

Comment:
nit: memory не обязательна. Вообще можно сделать еще лучше. Выделять память под масссив через ArrayPool:
```suggestion
var buffer = ArrayPool<byte>.Shared.Rent(256);
            var memory = new Memory<byte>(buffer[..256]);
```
Только надо не забыть освободить ресурсы, которые были выделены под массив. Их нужно будет вернуть обратно.
ArrayPool<byte>.Shared.Return(buffer); нужно будет вызвать в файнали блоке трай кетча. В мавлинке часто используется этот подход. В тестах должны быть примеры